### ISSUE TITLE
flatnode option need folder with write access for user renderer 

### ISSUE THREAD

**Issue by stevo01** [Overv/openstreetmap-tile-server#63]
I used following command for initial import of osm extract:
```
docker run \
	--name openstreetmap-tile-server-germany \
        --rm=false \
	-e THREADS=8 \
	-e UPDATES=enabled \
	-e AUTOVACUUM=off \
	-e OSM2PGSQL_EXTRA_ARGS="--flat-nodes /flat/flat_nodes.bin" \
	-v $PWD/volumes/download/germany-latest.osm.pbf:/data.osm.pbf \
	-v $PWD/volumes/download/germany.poly:/data.poly \
	-v openstreetmap-data-germany-latest:/var/lib/postgresql/10/main \
	-v openstreetmap-rendered-tiles-germany-latest:/var/lib/mod_tile \
	-v openstreetmap-flat-germany-latest:/flat \
	openstreetmap-tile-server-germany  \
	import
```
The import failed because folder flat/ is not writable for user renderer.
I used following patch to solve the problem:
```
diff --git a/run.sh b/run.sh
index 857d847..a5e90ce 100755
--- a/run.sh
+++ b/run.sh
@@ -51,6 +51,8 @@ if [ "$1" = "import" ]; then
         sudo -u renderer cp /data.poly /var/lib/mod_tile/data.poly
     fi
 
+    sudo chown renderer:renderer /flat
+
     # Import data
     sudo -u renderer osm2pgsql -d gis --create --slim -G --hstore --tag-transform-script /home/renderer/src/openstreetmap-carto/openstreetmap-carto.lua -C 2048 --number-processes ${THREADS:-4} ${OSM2PGSQL_EXTRA_ARGS} -S /home/renderer/src/openstreetmap-carto/openstreetmap-carto.style /data.osm.pbf
```

**Comment by Overv** [comment]
Nice. Could you submit this as a pull request as well? I could apply the patch but I'd like to properly credit you :)

**Comment by stevo01** [comment]
solved with a2eceb6bb50590186673537cc05fe6099708d2f2
